### PR TITLE
chore(package): bump dependencies

### PR DESCRIPTION
**Pull request by wizzy25**
#### Summary

bumps dependencies

drops support for node v4

### CODE REVIEW

File: src/spec/helpers/sftp.spec.coffee [hunk 1]
```diff
@@ -2,7 +2,6 @@ debug = require('debug')('spec:sftp')
 _ = require 'underscore'
 Promise = require 'bluebird'
 fs = Promise.promisifyAll require('fs')
-Connection = require 'ssh2'
```
Comment (PhilippSpo) [inline src/spec/helpers/sftp.spec.coffee]: Why don't we need this anymore?

File: src/coffee/helpers/user-agent.coffee [hunk 2]
```diff
@@ -3,7 +3,7 @@ getPkg = require 'load-module-pkg'
 
 module.exports = (pkgName, pkgVersion) =>
   throw new Error 'Package name and version is required' unless pkgName and pkgVersion
-  sdkPkgVersion = getPkg('sphere-node-sdk').version
+  sdkPkgVersion = getPkg.sync('sphere-node-sdk').version
```
Comment (PhilippSpo) [inline src/coffee/helpers/user-agent.coffee]: Is this also related to some version update?

Comment (wizzy25) [reply src/coffee/helpers/user-agent.coffee]: Version `1.0.0` of the [load-module-pkg](https://www.npmjs.com/package/load-module-pkg#sync) expects a callback with the old API. I used the new synchronous API here.

File: src/spec/helpers/sftp.spec.coffee (hunk 1 above)
Comment (wizzy25) [reply src/spec/helpers/sftp.spec.coffee]: Not used anywhere in this file
